velox/vector/arrow/Bridge.h

```diff
@@ -36,6 +36,7 @@ struct ArrowOptions {
   bool flattenDictionary{false};
   bool flattenConstant{false};
   TimestampUnit timestampUnit = TimestampUnit::kNano;
+  bool exportToStringView = false;
```

Comment:
can you add a documentation comment explaining what this does? I see we are already missing for the ones above :(

Reply:
Done